src/Components/Artist/ArtistCollectionsRail/__tests__/ArtistCollectionsRail.test.tsx

```diff
@@ -64,13 +68,15 @@ describe("CollectionsRail", () => {
         .find(ArrowButton)
         .at(1)
         .simulate("click")
-
-      expect(dispatch).toBeCalledWith({
-        action_type: "Click",
-        context_module: "CollectionsRail",
-        context_page_owner_type: "Artist",
-        subject: "clicked next button",
-        type: "Button",
+      // Settimeout needed here for carousel render
+      setTimeout(() => {
```

Comment:
This still needed with the `await` up above?

Reply:
This one I couldn't get to work with the await :(